### PR TITLE
packaging: fix packaging

### PR DESCRIPTION
**Pull request by oliver-sanders**
* comments appear to be invalid in the include_package_data config
  causing the config to be ignored

### CODE REVIEW

**Review by kinow — COMMENTED** [review]
Was going to test it but realized it's still draft. `MANIFEST.in` has some conflicts I think though.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-include py.typed
-include logging_config.json
-include logo.svg
-recursive-include cylc/uiserver/ui
+include cylc/uiserver/py.typed
+include cylc/uiserver/logging_config.json
+include cylc/uiserver/logo.svg
+recursive-include cylc/uiserver/ui/ *
```
Comment (kinow) [inline MANIFEST.in:4]: Missing newline, but can be fix later.

**Review by kinow — APPROVED** [review]
Did a `pip install build`, followed by `python -m build --sdist` using `master`. Then again with this PR. Immediately noticed the `py.typed` added to the file. I think the logo came from somewhere, and the ui files were included too.

Looks good to me :+1:

File: MANIFEST.in (hunk 1 above)
Comment (oliver-sanders) [reply MANIFEST.in:4]: Dammit, GH editor.

**Review by wxtim — APPROVED** [review]

**Pull request merged**